arches/app/views/base.py

```diff
@@ -130,4 +130,19 @@ def get_context_data(self, **kwargs):
         context["resource_map_layers"] = resource_layers
         context["resource_map_sources"] = resource_sources
 
+        all_map_layers = models.MapLayer.objects.all()
+        all_map_sources = models.MapSource.objects.all()
+
+        map_layers = []
+
+        if self.request.user.is_superuser is True:
+            map_layers = all_map_layers
+        else:
+            for m in all_map_layers:
+                if (m.addtomap is True and m.isoverlay is False) or (not self.request.user.has_perm("no_access_to_maplayer", m)):
```

Comment:
@aj-he This is now amended so that no overlays are shown if the user is in a group with no_access_to_maplayer set for that overlay, regardless if it is added to the map by default.  Does this solve the issue you raised?